infra/src/main/java/com/redhat/service/smartevents/infra/api/APIConstants.java

```diff
@@ -6,44 +6,49 @@
 public class APIConstants {
```

Comment:
(side note) I think some constants in this class can be moved to "core" constants to be re-used in `v1` and `v2` services. On the other side I think it's good to do it in another PR so to keep this one scoped

Reply:
Done.